### ISSUE TITLE
Add support for  --experimentalDecorators compiler flag

### ISSUE THREAD

**Issue by bryanmig** [ivogabe/gulp-typescript#180]
Decorators landed in 1.5 but throw compiler warnings unless the  --experimentalDecorators flag is passed in, as per the following message

```: error TS1219: Experimental support for decorators is a feature that is subject to change in a future release. Specify '--experimentalDecorators' to remove this warning.```

Tried passing `{experimentalDecorators: true}` in the `createProject()` method but it did not pass that along to the compiler.

**Comment by ivogabe** [comment]
It should work by setting `experimentalDecorators: true` in your gulpfile or tsconfig file. Given that you've already tried that, can you post your gulpfile? Or did you misspell `experimentalDecorators`?

**Comment by bryanmig** [comment]
This was my mistake.  I unknowingly had two different gulp pipelines and only added the flag to one of them.  The other was emitting the warnings.  Thanks for the quick response.
